Exercise 21-JS/index.js

```diff
@@ -0,0 +1,5 @@
+"use strict"
+alert("Welcome, Enter two numbers to see if the first is greater than the second");
+let firstNumber=prompt("Enter First Number");
+let secondNumber=prompt("Enter Second Number");
```

Comment:
As a best practice, when you are comparing numbers make sure to convert them to numbers first if they are not already. In this case, the `firstNumber` and `secondNumber` variables are not numbers.